docs/examples/http.py

```diff
@@ -0,0 +1,12 @@
+from nameko.web import http
+
+
+class Service(object):
+
+    @http('GET', '/get/<int:value>')
+    def get_method(self, value):
+        return {'value': value}
+
+    @http('POST', '/post')
+    def do_post(self, value):
```

Comment:
I thought the values were unpacked, but it looks like I was wrong.

@davidszotten was this the original implementation?